Assets/scripts/FilterOpen.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class FilterOpen : MonoBehaviour
+{
+    public GameObject panelToOpen;
+    public Text textExtract;
+    public Text textChange;
+
+    public void OpenPanel()
+    {
+        if (panelToOpen != null)
+        {
+            bool isActive = panelToOpen.activeSelf;
+
+            panelToOpen.SetActive(!isActive);
+        }
+    }
+
+    public void showAny()
+    {
+        LoadCatalog scriptToAccess = GameObject.Find("Viewport").GetComponent<LoadCatalog>();
+
+        scriptToAccess.resetListing();
+    }
+
+    public void showCategories(string categoryName)
+    {
+        LoadCatalog scriptToAccess = GameObject.Find("Viewport").GetComponent<LoadCatalog>();
```

Comment:
I think  the variable name is not clear for me. Something like CatalogScene or something similar would be more clear.

Reply:
Ok I'll rename them later